src/AzureSign.Core/AuthenticodeKeyVaultSigner.cs

```diff
@@ -96,6 +97,23 @@ static char[] NullTerminate(ReadOnlySpan<char> str)
                 flags |= SignerSignEx3Flags.SPC_EXC_PE_PAGE_HASHES_FLAG;
             }
 
+            if (appendSignature)
+            {
+                // OperatingSystem.IsWindowsVersionAtLeast is not yet introduced in .Net Standard 2.0
```

Comment:
I think this version check is technically wrong but would probably work in practice. This would return false if major returned `42` but build returned `3000`. Presumably Windows 42.0.3000 is newer than 10.0.22000. I don't know if it is written in stone anywhere that the build number is forever incrementing.

I think you should have a static field like `_win11 = new Version(10, 0, 22000)` and do `Environment.OSVersion >= _win11` and just let the version class take care of it.